### PR TITLE
Added Car_Game

### LINKED ISSUE #40: Car_Game
**Is your feature request related to a problem? Please describe.**
I would like to add Car Game

**Describe the solution you'd like**
Built using HTML, CSS, and Javascript.

**Additional context**
Will be adding comments and screenshots of the preview in the Readme file.

### PR DESCRIPTION
**Pull request by thejaswin123**
Fixes #40 
 
## Description
Built using HTML, CSS, and Javascript. This is a Basic Car game where the player wins if car reaches end of track without hitting on any other cars. Player can use left and right arrow keys to move the car to left and right, and up and down arrow keys to speed up and slow down. Player can start the game by pressing the Start Game button.

### CODE REVIEW

**Review by kanishkaa24 — CHANGES_REQUESTED** [review]
Kindly add the CSS and JS file to static and html file in the main directory with the game name and not index.html.

**Comment by thejaswin123** [comment]
> Kindly add the CSS and JS file to static and html file in the main directory with the game name and not index.html.

Changes done

**Comment by kanishkaa24** [comment]
> > Kindly add the CSS and JS file to static and html file in the main directory with the game name and not index.html.
> 
> Changes done

Changes that must be addressed:

- Add the HTML file in the root directory (of the repository) and CSS, JS, and assets to the ```static folder``` of the root directory. This is required to deploy to gh-pages. Do not create your own folder.
- Add a card on the homepage of the website to access this game.
- Add a navigation in your game HTML file to the homepage of the website.

**Comment by thejaswin123** [comment]
> > > Kindly add the CSS and JS file to static and html file in the main directory with the game name and not index.html.
> > 
> > 
> > Changes done
> 
> Changes that must be addressed:
> 
> * Add the HTML file in the root directory (of the repository) and CSS, JS, and assets to the `static folder` of the root directory. This is required to deploy to gh-pages. Do not create your own folder.
> * Add a card on the homepage of the website to access this game.
> * Add a navigation in your game HTML file to the homepage of the website.
> * Stash all the commits into a single commit.

Changes made

File: Car_game.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Car Game</title>
+    <link rel="stylesheet" href="car_game.css">
+</head>
+
+<body>
+    <div class="btnStart"> Start Game</div>
```
Comment (kanishkaa24) [inline Car_game.html:12]: Add a navigation to the main homepage. The link should direct you back to the JS Games website.

**Review by kanishkaa24 — CHANGES_REQUESTED** [review]

File: Car_game.html (hunk 1 above)
Comment (thejaswin123) [reply Car_game.html:30]: navigation bar added that redirects to JS Games website on click

**Review by kanishkaa24 — APPROVED** [review]

**Review by kanishkaa24 — APPROVED** [review]

**Review by geekymeeky — CHANGES_REQUESTED** [review]
It's not working. Kindly check it.

**Comment by thejaswin123** [comment]
> It's not working. Kindly check it.

Now it should work, changes are made.

**Comment by kanishkaa24** [comment]
Screenshot (400)
See your game is not visible on the site.

**Comment by thejaswin123** [comment]
> Screenshot (400)
> See your game is not visible on the site.

how to add the card of the game?

**Comment by thejaswin123** [comment]
Added in index.html kindly please check
Screenshot 2021-07-14 194001

**Review by geekymeeky — APPROVED** [review]

**Pull request merged**